thavalon-server/src/connections/account_handlers.rs

```diff
@@ -136,6 +147,16 @@ pub async fn handle_add_user(
     log::info!("Successfully added user to the database.");
     let (jwt, refresh_token) = token_manager.create_jwt(&player_id).await;
     let response = create_validated_response(jwt, refresh_token, StatusCode::CREATED).await;
+    if let Err(e) = account::send_email_verification(&new_user.email).await {
+        log::error!(
+            "ERROR: failed to send account verification email to {}. {}.",
+            player_id,
+            e
+        );
+        // TODO: Support resending email verification. We don't want to return
+        // an error, since the account was created succesfully, but we need to
+        // resend this email somehow.
```

Comment:
We could have a background task for sending verification emails that reads off a channel. Then, if sending fails it can resubmit to the channel

Reply:
Yeah I more meant that todo for the frontend. I created #40 for this.